lab_1_keywords_tfidf/main.py

```diff
@@ -2,22 +2,35 @@
 Lab 1
 Extract keywords based on frequency related metrics
 """
-from typing import Optional, Union
+from typing import Optional, Union, List
+import re
 
 
-def clean_and_tokenize(text: str) -> Optional[list[str]]:
+
+def clean_and_tokenize(text: str) -> Optional[List[str]]:
```

Comment:
students are not supposed to change the lines where function is defined, discard this change

Reply:
Done.